db/schema.rb

```diff
@@ -465,6 +465,12 @@
     t.datetime "created_at"
     t.datetime "updated_at"
     t.string   "picture"
+    t.boolean  "payed",                default: false
+    t.boolean  "swags",                default: false
+    t.boolean  "swags_received"
+    t.string   "company_address"
```

Comment:
I think just address should be sufficient (less complicated / easier to remember)